app/services/attribute_asserter.rb

```diff
@@ -98,6 +104,20 @@ def wrap_with_zipcode_formatter(getter)
     end
   end
 
+  def wrap_with_dob_formatter(getter, american_date_format:)
+    proc do |principal|
+      if (date_str = getter.call(principal))
```

Comment:
It's good to an assignment in an if clause again...the old linter used to nail me for this!